### PR TITLE
Fix: fix logo

### LINKED ISSUE #136: Fix logo
Fix logo paddings

### PR DESCRIPTION
**Pull request by stroich**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

### Actions Completed:
1. Removed duplicate logo icons.
2. Implemented SVG component

## Related Tickets & Documents

- Related Issue #136 
- Closes #136 

## Screenshots, Recordings
2024-04-28_19h56_29

2024-04-28_19h56_41

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

File: src/app/components/logo-wrapper/logo-wrapper.tsx [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
-
 import './logo-wrapper.scss';
 
 interface LogoWrapperProps {
   type: 'navbar' | 'footer';
 }
 
 export const LogoWrapper = ({ type }: LogoWrapperProps) => {
-  const logo = type === 'navbar' ? rsLogoNavbar : rsLogoFooter;
+  const color = type === 'navbar' ? '#000000' : '#ffffff';
```
[the remark this answers is not in the document]
Comment (ansivgit) [reply src/app/components/logo-wrapper/logo-wrapper.tsx:8]: I suggest to use constants for this from `_constants.scss`

[the remark this answers is not in the document]
Comment (stroich) [reply src/app/components/logo-wrapper/logo-wrapper.tsx:8]: To use variables from scss, we have to use a modular system. I'm not sure it's worth doing this for the sake of two variables. I agree with the rest, I will correct it.

File: src/app/components/logo-wrapper/logo-wrapper.tsx
[the remark this answers is not in the document]
Comment (stroich) [reply src/app/components/logo-wrapper/logo-wrapper.tsx:1]: I fixed this bug

File: src/app/components/logo-wrapper/logo-wrapper.tsx [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
+            fill="#FFB749"
+          />
+          <path
+            d="M39.4768 35.089L33 35.4666C33.1262 37.3671 34.0021 39.16 35.4636 40.5088C36.9117 41.8323 39.5076 42.4941 43.2515 42.4941C46.3564 42.5823 49.9058 41.8146 51.821 39.1569C52.5929 38.0934 53.0033 36.8427 52.9998 35.564C53.0217 33.1848 51.4339 31.2297 49.3044 30.3147C47.2632 29.4766 45.1198 28.8674 42.9204 28.5C42.1107 28.41 41.3327 28.1563 40.6423 27.757C39.9039 27.2597 40.078 26.2272 40.735 25.7596C42.6084 24.5207 45.6299 25.5545 45.8608 27.9032L52.2845 27.5621C52.1703 25.768 51.1844 24.0545 49.6356 22.9583C47.6987 21.8887 45.4532 21.3874 43.1986 21.5212C41.3493 21.4527 39.5037 21.7218 37.7682 22.3128C35.6082 23.1125 33.829 25.064 33.8344 27.4525C33.7931 28.9377 34.5158 30.4088 35.755 31.3621C37.6454 32.6238 39.8325 33.4582 42.139 33.798C43.3833 33.9637 44.5727 34.3795 45.6224 35.0159C46.5878 35.7309 46.5807 37.167 45.5959 37.8903C44.5078 38.6532 42.9034 38.7416 41.6818 38.2468C40.3717 37.716 39.6048 36.4784 39.4768 35.089Z"
+            fill="#FFB749"
+          />
+        </g>
+        <defs>
+          <clipPath id="clip0_5701_38384">
```
[the remark this answers is not in the document]
Comment (stroich) [reply src/app/components/logo-wrapper/logo-wrapper.tsx]: The component has been created

**Review by ansivgit — APPROVED** [review]

**Review by Quiddlee — APPROVED** [review]

**Pull request merged**